applications/plugins/CGALPlugin/src/CGALPlugin/DecimateMesh.inl

```diff
@@ -125,7 +125,7 @@ void DecimateMesh<DataTypes>::doUpdate()
     }
     else
     {
-        msg_erro() << "You must add a stop condition using either targetedNumberOfEdges or targetedRatioOfEdges" ;
+        msg_error(this) << "You must add a stop condition using either targetedNumberOfEdges or targetedRatioOfEdges" ;
```

Comment:
The "this" is not needed. 
msg_error() is equivalent.

Reply:
Hi, I think I had an error without adding "this". I don't know if it was related to an include problem. Maybe you are wright it's not necessary. Thx :)

Reply:
no I think, you just got a compilation error because it was msg_erro   (r missing)